plugins/in_head/in_head.c

```diff
@@ -255,81 +255,29 @@ static int in_head_collect(struct flb_input_instance *i_ins,
 static int in_head_config_read(struct flb_in_head_config *ctx,
                                struct flb_input_instance *in)
 {
-    const char *filepath = NULL;
-    const char *pval = NULL;
-
-    /* filepath setting */
-    filepath = flb_input_get_property("file", in);
-    if (!filepath) {
+    int ret;
+    /* Load the config map */
+    ret = flb_input_config_map_set(in, (void *)ctx);
+    if (ret == -1) {
```

Comment:
Could you output log ?
If `flb_input_config_map_set` failed, fluent-bit will exit without error logs.